tests/test_features/test_radviz.py

```diff
@@ -16,13 +16,17 @@
 ##########################################################################
 ## Imports
 ##########################################################################
-
+import unittest
 import numpy.testing as npt
 
 from tests.base import VisualTestCase
 from tests.dataset import DatasetMixin
 from yellowbrick.features.radviz import *
 
+try:
+    import pandas
```

Comment:
I noticed that in some of the tests, such as [test_importances.py](https://github.com/DistrictDataLabs/yellowbrick/blob/develop/tests/test_features/test_importances.py), that we use ``import pandas as pd`` instead of ``import pandas``. Of course, they both work; however, I thought this might be another issue we could add to the discussion about establishing a uniform style.

Reply:
@Kautumn06 thanks for pointing this out.

I've created an issue around this style and convention issue : https://github.com/DistrictDataLabs/yellowbrick/issues/340

In this particular case, I'm not as concerned with uniform conventions in the test suite as it is a little deeper in the library, but the point still is well taken.